test/bin/github-action.test.js

```diff
@@ -0,0 +1,75 @@
+const { main } = require("../../src/bin/main");
+jest.mock("../../src/bin/main");
+const { getInput } = require("@actions/core");
+jest.mock("@actions/core");
+const { start } = require("../../src/bin/github-action");
+
+afterEach(() => {
+  jest.clearAllMocks();
+});
+
+describe("start", () => {
+  test("no inputs", () => {
+    // Act
+    start();
+
+    // Assert
+    expect(main).toBeCalledTimes(1);
+    expect(main).toHaveBeenCalledWith({
+      artifacts: undefined,
+      debug: false,
+      exclude: undefined,
+      filter: undefined,
+      logFilePath: undefined,
+      outputFile: undefined,
+      printModuleList: false,
+      skipOutput: false
+    });
+  });
+
+  test("log-file-path input", () => {
+    // Arrange
+    getInput.mockReturnValueOnce("log-file-path");
+
+    //Act
+    start();
+
+    // Assert
+    expect(main).toBeCalledTimes(1);
+    expect(main).toHaveBeenCalledWith(
+      expect.objectContaining({
+        logFilePath: "log-file-path"
+      })
+    );
+  });
+
+  test("all inputs", () => {
+    // Arrange
+    getInput.mockReturnValueOnce("log-file-path");
+    getInput.mockReturnValueOnce("output-file");
+    getInput.mockReturnValueOnce(true);
+    getInput.mockReturnValueOnce("filter");
+    getInput.mockReturnValueOnce("exclude");
+    getInput.mockReturnValueOnce("artifacts");
+    getInput.mockReturnValueOnce(false);
```

Comment:
remember getInput only returns string
```suggestion
    getInput.mockReturnValueOnce("false");
```